src/tuner/camera.py

```diff
@@ -0,0 +1,57 @@
+import cv2
+
+class Camera:
+    
+    def __init__(self):
+        """
+        Constructor
+        """
+        self.index = 0
+        self.max_index = 1
+        self.cap = None
```

Comment:
We should type hint what cap is since it is a special object

Since we are doing python 3.11+ we could do
```suggestion
        self.cap: cv2.VideoCapture | None = None
```

But for compatibility for python 3.10 we could instead do
```suggestion
        self.cap: Optional[cv2.VideoCapture] = None
```

Reply:
Wow, completely different syntaxes between versions of a language?  How did it become so popular with all of the breaking backwards compatility?

Reply:
Neither suggestion worked:
self.cap = cv2.VideoCapture | None = None
resulted in 
SyntaxError: cannot assign to expression

and
self.cap = Optional[cv2.VideoCapture] = None
resulted in
TypeError: '_SpecialForm' object does not support item assignment

Reply:
Hmmmm.... there is a way to do it that should be found -- bad Gemini